tests/RedisConfigs/Sentinel/redis-7010.conf

```diff
@@ -1,4 +1,6 @@
 port 7010
+#requirepass changeme
+#masterauth changeme
```

Comment:
Can we revert changes to these files please?

Reply:
So the reason why I left those in there is twofold. 1. It was hard to figure out all the places that you have to set a password in a master/slave with sentinel setup. 2. The Windows port of Redis running in sentinel mode doesn’t work with a password so I had to turn it off in order for tests to pass. Also, having the password left in the code didn’t seem to negatively affect anything so I left it there so that’s it’s just a matter of commenting out all the password config entries to be able to test using a password in the setup.

Reply:
IMO, it serves more to confuse right now than it makes easy, so I'd vote to remove them. Right now the code config says you need a password, the redis config says you didn't, so the logic question anyone coming here is going to ask is "...why is this commented out?", and "why does this code still work?" - give the aim isn't to actually test password auth here (we're doing that separately), can we tidy and remove the confusion?